migrations/20170407090713_user_voting_bias_table.js

```diff
@@ -0,0 +1,31 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('voter_biases', function(table) {
+    table.increments('id').primary().index();
+    table.float('bias').notNullable().defaultTo(0.0);
+
+    table.integer('team_id').unsigned().notNullable().index();
+    table.foreign('team_id')
+      .references('id')
+      .inTable('teams')
+      .onDelete('CASCADE')
+      .onUpdate('CASCADE');
+
+    table.integer('user_id').unsigned().notNullable().index();
+    table.foreign('user_id')
+      .references('id')
+      .inTable('users')
+      .onDelete('CASCADE')
+      .onUpdate('CASCADE');
+  })
+  .raw(`
+    CREATE UNIQUE INDEX
+      only_one_bias_per_team
+    ON
+      voter_biases(user_id, team_id)
+  `);
```

Comment:
Users accumulated score will be transferred to that team. Bias value is not affected by users current guild.